pandas/tests/resample/test_base.py

```diff
@@ -112,6 +112,19 @@ def test_resample_empty_series(freq, empty_series, resample_method):
     tm.assert_series_equal(result, expected, check_dtype=False)
 
 
+@pytest.mark.parametrize("freq", ["M", "D", "H"])
+@pytest.mark.parametrize("resample_method", ["count", "size"])
+def test_resample_count_empty_series(freq, resample_method):
```

Comment:
can you use the empty_series fixture which runs for each of DTI, TDI, and PI

Reply:
Done